common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -850,7 +850,7 @@ private AuthenticationConstants() {
         /**
          * The value for pkeyauth redirect.
          */
-        public static final String PKEYAUTH_REDIRECT = "urn:http-auth:PKeyAuth";
+        public static final String PKEYAUTH_REDIRECT = "urn:http-auth:pkeyauth";
```

Comment:
My bad, when i searched the usages for constant I accidentally did in common repo instead of ADAL repo and assumed it was not used, Thanks @iambmelt for catching this